app/services/store_sp_metadata_in_session.rb

```diff
@@ -58,7 +44,6 @@ def update_session
       request_url: sp_request.url,
       request_id: sp_request.uuid,
       requested_attributes: sp_request.requested_attributes,
-      aal_level_requested: aal_level_requested_value,
```

Comment:
We actually need to leave this value behind and remove it in a separate commit. During the deploy the hosts that a running old code will expect this to be here.

Reply:
Put back.